Automattic/vip-cli
# Upgrading statsd container to 0.9.0

## Description

This PR bumps the version of the `statsd` container that's being used, from `0.8.6` to `0.9.0`.

## Steps to Test

1. Check out PR.
2. Rebuild a dev-environment.
3. Run `docker logs -f {STATS CONTAINER ID}`
4. Run `vip dev-env exec -- wp vip-search index --setup` and expect to see some logs.
